From: David Teigland <teigland@redhat.com>
Subject: [RHEL5 PATCH] dlm: disable debugging output
Date: Wed, 3 Jan 2007 15:32:18 -0600
Bugzilla: 221326
Message-Id: <20070103213218.GA5902@redhat.com>
Changelog: dlm: disable debugging output


bz 221326

Disable the verbose debugging output that the dlm is currently printing.

Index: linux-2.6.19-git19-quilt/fs/dlm/dlm_internal.h
===================================================================
--- linux-2.6.19-git19-quilt.orig/fs/dlm/dlm_internal.h	2006-12-14 14:25:32.000000000 -0600
+++ linux-2.6.19-git19-quilt/fs/dlm/dlm_internal.h	2006-12-18 16:24:03.000000000 -0600
@@ -69,7 +69,6 @@
 #define log_error(ls, fmt, args...) \
 	printk(KERN_ERR "dlm: %s: " fmt "\n", (ls)->ls_name , ##args)
 
-#define DLM_LOG_DEBUG
 #ifdef DLM_LOG_DEBUG
 #define log_debug(ls, fmt, args...) log_error(ls, fmt, ##args)
 #else

